### PR TITLE
[`pylint`]: bidirectional-unicode

### PR DESCRIPTION
**Pull request by colin99d**
Ref: #970

### CODE REVIEW

File: src/rules/pylint/rules/bidirectional_unicode.rs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    '\u{2069}', //{POP DIRECTIONAL ISOLATE}
+    // The following was part of PEP 672:
+    // https://www.python.org/dev/peps/pep-0672/
+    // so the list above might not be complete
+    '\u{200F}', //{RIGHT-TO-LEFT MARK}
+                // We don't use
+                //   "\u200E" # \n{LEFT-TO-RIGHT MARK}
+                // as this is the default for latin files and can't be used
+                // to hide code
+];
```
Comment (charliermarsh) [inline src/rules/pylint/rules/bidirectional_unicode.rs:28]: What's the source for this list?

Comment (colin99d) [reply src/rules/pylint/rules/bidirectional_unicode.rs:28]: https://github.com/PyCQA/pylint/blob/main/pylint/checkers/unicode.py#L37

Comment (charliermarsh) [reply src/rules/pylint/rules/bidirectional_unicode.rs:28]: Thank you :)

**Comment by colin99d** [comment]
Screenshot 2023-02-06 at 9 51 41 PM
Looks like Github doesn't like it either!

**Comment by colin99d** [comment]
Also, the file `src/registry.rs` should be deleted, right?

**Comment by charliermarsh** [comment]
Yes although I'm wondering if we lost any rules in the refactor that moved that file.

**Comment by colin99d** [comment]
When I tried to add a rule there on accident it threw a bunch of errors, I would imagine that would happen with other errors as well.

**Pull request merged**